cmd/crictl/main.go

```diff
@@ -105,7 +120,27 @@ func main() {
 	}
 
 	app.Before = func(context *cli.Context) error {
-		if context.GlobalBool("debug") {
+		configFile := context.GlobalString("conf")
+		config, err := ReadConfig(configFile)
+		if err != nil {
+			return cli.NewExitError("Failed to parse config file", 1)
```

Comment:
Should make this optional. e.g. if no configFile is provided, it should continue to figure out params via flags.